joomla/joomla-cms
# [4.1] Finder auto-suggestions broken

### Summary of Changes

Finder auto-suggestions for search fields are broken due to encoded URL for suggestion requests.

### Testing Instructions

Load finder page, enable suggestions, try to search for any existing word, see requests with encoded `&amp;` and invalid data.

### Actual result BEFORE applying this Pull Request

Finder suggestions are broken.

### Expected result AFTER applying this Pull Request

Finder suggestions are fine.

### Documentation Changes Required

No.
